reporting_endpoint.py

```diff
@@ -1,5 +1,6 @@
 # pylint: disable=unbalanced-tuple-unpacking
 # pylint: disable=too-many-locals
+# pylint: disable=unnecessary-lambda-assignment
```

Comment:
update from master and pylint no longer used

Reply:
this should no longer be there after the rebase thing i did to merge.